libs/hdf-converters/src/xccdf-results-mapper.ts

```diff
@@ -278,142 +299,149 @@ export class XCCDFResultsMapper extends BaseConverter {
         status: 'loaded',
         controls: [
           {
-            path: 'Benchmark.Group',
+            path: 'Benchmark',
+            pathTransform: getRulesInBenchmark,
             key: 'id',
             tags: {
```

Comment:
![image](https://github.com/mitre/heimdall2/assets/2015821/417cba1f-3c12-4376-ad50-01bf7e967613)

warning should be added as a conditionally supplied tag or subdescription